src/c++/perf_analyzer/client_backend/triton/triton_client_backend.cc

```diff
@@ -32,21 +33,36 @@ namespace triton { namespace perfanalyzer { namespace clientbackend {
 namespace tritonremote {
 //==============================================================================
 
+struct triton::client::HttpSslOptions ParseHttpSslOptions(
+    const struct triton::perfanalyzer::clientbackend::SslOptionsBase&);
+std::pair<bool, struct triton::client::SslOptions> ParseGrpcSslOptions(
+    const struct triton::perfanalyzer::clientbackend::SslOptionsBase&);
+
 Error
 TritonClientBackend::Create(
     const std::string& url, const ProtocolType protocol,
+    const SslOptionsBase& ssl_options,
     const grpc_compression_algorithm compression_algorithm,
     std::shared_ptr<Headers> http_headers, const bool verbose,
     std::unique_ptr<ClientBackend>* client_backend)
 {
   std::unique_ptr<TritonClientBackend> triton_client_backend(
       new TritonClientBackend(protocol, compression_algorithm, http_headers));
   if (protocol == ProtocolType::HTTP) {
+    struct triton::client::HttpSslOptions http_ssl_options =
+        ParseHttpSslOptions(ssl_options);
     RETURN_IF_TRITON_ERROR(tc::InferenceServerHttpClient::Create(
-        &(triton_client_backend->client_.http_client_), url, verbose));
+        &(triton_client_backend->client_.http_client_), url, verbose,
+        http_ssl_options));
   } else {
+    std::pair<bool, struct triton::client::SslOptions> grpc_ssl_options_pair =
+        ParseGrpcSslOptions(ssl_options);
+    bool use_ssl = grpc_ssl_options_pair.first;
+    struct triton::client::SslOptions grpc_ssl_options =
+        grpc_ssl_options_pair.second;
```

Comment:
remove `struct`

Reply:
good catch--removed here and everywhere else applicable 👍